### DISCUSSION TITLE
Docker?

### DISCUSSION THREAD

**Discussion by Darren120** [remix-run/remix#693] [Q&A] [answered]
Is remix easy to put on docker? its kinda complicated with nextjs. 

also is free?

**Comment by matmilbury** [comment, marked as answer]
Putting Remix on docker is easy. You can, for example, see how @kentcdodds did this with his website [here](https://github.com/kentcdodds/kentcdodds.com/blob/b42c56b8455a0387c1b3181311dd10c88d1648ee/Dockerfile).

Remix is released under the [MIT license](https://github.com/remix-run/remix/blob/main/LICENSE.md) which means you don't have to pay for using it.
